src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape:Shape;
+  color:Color;
+  getArea():number;
```

Comment:
```suggestion
  shape: Shape;
  color: Color;
  getArea(): number;
```
please check code style in all cases